src/app/clusters/basic-information/BasicInformationCluster.cpp

```diff
@@ -45,10 +41,36 @@ using namespace chip::app::Clusters;
 using namespace chip::app::Clusters::BasicInformation;
 using namespace chip::app::Clusters::BasicInformation::Attributes;
 using namespace chip::DeviceLayer;
-using chip::Protocols::InteractionModel::Status;
 
 namespace {
 
+// fixed size for location strings
+static constexpr size_t kFixedLocationLength = 2;
```

Comment:
Why do we have both this and kExpectedFixedLocationLength?  At the very least that should be documented... but I think we should only have one constant for this constant, honestly.  Keep the one that was already here: kExpectedFixedLocationLength.